css/src/components/checkbox.scss

```diff
@@ -0,0 +1,114 @@
+/* stylelint-disable selector-max-specificity, max-nesting-depth, selector-no-qualifying-type */
+
+$checkbox-border-radius: $radius-sm !default;
+$checkbox-border-width: 0.0625em !default;
+$checkbox-unchecked-border-color: $text !default;
+$checkbox-color: $primary !default;
+$checkbox-check-color: $primary-invert !default;
+$checkbox-check-hover-color: $text-subtle !default;
+$checkbox-size: 1.25em !default;
+$checkbox-check-height: 0.75em !default;
+$checkbox-check-top: 0.1875em !default;
+$checkbox-check-width: 0.3125em !default;
+$checkbox-check-thickness: 0.0625em !default;
+$checkbox-timing-function: $input-timing-function !default;
+$checkbox-duration: $input-transition-duration !default;
+$checkbox-spacing: $spacer-3 !default;
+
+@mixin checkmark {
+	display: block;
+	position: absolute;
+	top: $checkbox-check-top;
```

Comment:
This requirement sneakily slid in during this PR, but we want to do logical properties whenever possible, so `top` becomes `inset-block-start`.

Reply:
Updated